build-support/aws/ec2/packer/jenkins-slave.customize.sh

```diff
@@ -50,7 +50,7 @@ users:
 - default
 - name: jenkins
   ssh_authorized_keys:
-  - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCWdCCh5LYbwKgFaEO2s1Hr+ONn55R+ViwDtCcLYGCdXMGFOD/0CEPYAPW2D2Cn29r1bxKryIGyA4RsAR5Op12AI9GwTXJF9nynjkeV+gKj2EMnqHzw53r1Rc5j8HMSTRXtjoRapjQ/BzNMHOkTVxwqdqnNmKlvRcFCpMCUA7Xz12q/4bhAHJ/TVnfZIDYBxNbvDfg8wKp0OHwnt2URbZJjKHll95wTJuiu72YoydiHEiCE2+Ja8l7/V54V7S8zoaJrxMDJiHJe2Yfx658D/olr83pGISytBXbhlxFFeVKnmj1l0sUpiUJSxlu2rq0tZdUJr/CYLae4r6mNKIoxxXB1
+  - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDZV8wo8XzRkOWyhDPeozZMM0vwUFBAxxGFWMG7T5Jkezd6csJWLbXGARBci6MwNZJXbQGOs/6X9P9Ci/UHVZM8OZ6sPfRl6PRMao/RaZnj0rpS2A/hfdUDx7QAwWDILYkKKdKHmiC3/3H63Rtup/Ee/7wvzcUcT5oZPsGSNpP/33WGnbiY3CpcR1OAeo1nEiNMyivBWBw8PhJWGi0iCbwAlpduMX9Fws4ItPXQgwZkCs+65lbwIZSluzPkHq0YoLTnrWPbFg/KtNLLA4q/MYAFunUJOH9bb1ay9B+t97I8dMpie7lOXIu/N9b4EJrS+oXoSTteRw5Lp6pQJ/xHTUcv
```

Comment:
Can the old pub key be kept and just the old keypair imported into ec2?
The key on the left is also the key used by mac-mini1 and mac-mini2, so changing it would mean you need to go hand edit mac-mini1 & mac-mini2 again as well.